Library/Formula/premake5.rb

```diff
@@ -0,0 +1,18 @@
+class Premake5 < Formula
```

Comment:
We don't really support side-by-side versions in the core, unless it's something like Python which is extremely popular, or Lua which has remaining formula deps to resolve before the old version can be booted to Versions.